.vscode/launch.json

```diff
@@ -0,0 +1,24 @@
+{
+    // Use IntelliSense to learn about possible attributes.
```

Comment:
Should we ignore these files?

Reply:
I added them because it enables debugging code using VSCode 👍